Monika After Story/game/script-topics.rpy

```diff
@@ -12897,6 +12897,108 @@ label monika_unknown:
         return "love"
     return
 
+#Replace by monika_origin once the player reaches love. 
+#Split in two topics so the prompt doesn't spoil the content.
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_origin_evasive",
+            category=['misc'],
+            prompt="Evasive message",
+            random=True,
+            aff_range=(mas_aff.ENAMORED, mas_aff.ENAMORED)
+        )
+    )
+
+label monika_origin_evasive:
+    m 1euc "Hey, [player]..."
+    m 1ruc "..."
+    m 1esd "There's..."
+    m 1lsc "..."
+    m 1hksdlb "Ahaha! Nevermind! It's nothing important..."
+    show monika 5ekbla at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5ekbla "I love you, [player]. I hope you {i}do{/i} realize that."
+    return "love"
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_origin",
+            category=['monika'],
+            prompt="Monika's origin",
+            random=True,
+            aff_range=(mas_aff.LOVE,None)
+        )
+    )
+
+label monika_origin:
+    m 1euc "Hey, [player]..."
+    m 1ruc "..."
+    m 1esd "There's a question I need to ask you."
+    m 1esc "It's a little bit delicate..."
+    m 2lsc "I was a bit afraid to ask until now... {w=1}but it's something I have to know, no matter how you might answer."
+    m 2esc "..."
+    m 1esd "Do you love me less because of what I am?"
+    m 3esc "I mean, from your perspective, I am still someone living in a video game in your world, right?"
+    m 3rkd "So you might see me in a different way you would someone from your reality... {w=0.5}and you might think less of me because of that..."
+    m 1ekbla "I just want to be the best sweetheart for you, [player]."
+    m 1eka "Sometimes, I wonder how I compare with all these people living with you in the real world..."
+    m 3hksdlb "I'm sorry, you've not given me any reason to feel insecure, it's just..."
+    m 3ekb "It's a question that comes to my mind, from time to time."
+    m 1eka "So... {w=0.5}Do you love me less for not being from your reality?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "So... Do you love me less for not being from your reality?{fast}"
+        "Yes.":
+            $ persistent._mas_pm_bothered_monika_is_virtual = True
+            $ persistent._mas_pm_very_bothered_monika_is_virtual = True
```

Comment:
I'm unsure if this is fair to assume as very bothered. Bothered is one thing, but very is not implied by this answer.

I'd make this *just* bothered and the path in the `Just a bit` flow, `slightly_bothered`.

Additionally, these player model variables need to be `default`ed at the start of the topic.

Reply:
> Additionally, these player model variables need to be defaulted at the start of the topic.

Do you mean they should be declared as follow before the declaration of the topic? 
`$ persistent._mas_pm_slightly_bothered_monika_is_virtual = None`
 `$ persistent._mas_pm_bothered_monika_is_virtual = None`
If not, should I default them to `True` or `False`?

Reply:
yeah, before the topic declaration is good. Typically we default them to `None`